cli/commands/init.ts

```diff
@@ -55,13 +59,33 @@ async function generate(
         await writeFile(outputDir, contentPath, content, options);
       },
     },
+    canvas: {
+      create: createCanvas,
+    },
   };
 
   console.log(progress("Generating mod template..."));
+  await loadFont();
   await generator.generateTemplate(options);
   console.log(success("Done!"));
 }
 
+async function loadFont() {
+  const host = Deno.env.get("FABRIC_HOST") ?? "https://fabricmc.net";
+
+  const response = await fetch(
+    `${host}/assets/fonts/ComicRelief-Regular.woff2`,
+  );
```

Comment:
Can we bundle the font in the cli javascript file?

Reply:
Done, though it's a little hacky :P